### ISSUE TITLE
Issue with searching for cards on versionfd4f762

### ISSUE THREAD

**Issue by ShadowlordSetsu** [Cockatrice/Cockatrice#807]
I've noticed that since I updated to version Fd4f762 of the cockatrice client, my search isn't working  at all. is there a way to fix this or that someone could address this issue?

**Comment by Nightfirecat** [comment]
Related to #796?

**Comment by poixen** [comment]
We would need to reproduce it first. Can you describe the steps needed, provide a screenshot and tell us what OS you are on.

**Comment by ShadowlordSetsu** [comment]
I can't provide a screen shot as I went back to versionB06B818. but  I do use  Windows 8. Step by step.
1) I updated my client from woogerworks.com
2)Ran my oracle to make sure my cardbase was up to date
3)I opened up cockatrice like normal and logged in
4) in the center part of the deck editor where it allows you to search for names,types,color and cmc, set and  card text, I tried to search for cards with the word " damage" on them, but nothing happened. My card list on the far left stayed the same.
